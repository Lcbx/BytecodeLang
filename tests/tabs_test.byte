def a = 1
	
def b = 2
	def c = 3
def d = 4
def e = 5 # comment1
def f = 6
		def g = 7
		def h = 8
		def i = 9 # comment2
	
	# comment 3