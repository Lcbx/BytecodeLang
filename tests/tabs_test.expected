tokenize_Out:
NAME: def, NAME: a, OP: =, INT: 1, NAME: def, NAME: b, OP: =, INT: 2, TABS: 1, NAME: def, NAME: c, OP: =, INT: 3, NAME: def, NAME: d, OP: =, INT: 4, NAME: def, NAME: e, OP: =, INT: 5, NAME: def, NAME: f, OP: =, INT: 6, TABS: 2, NAME: def, NAME: g, OP: =, INT: 7, TABS: 2, NAME: def, NAME: h, OP: =, INT: 8, TABS: 2, NAME: def, NAME: i, OP: =, INT: 9, EOF: line 11
__SECTION__
tokenize_Err:

__SECTION__
compile_Out:
Error line 3, column 1 : illegal token : 'TABS: 1'
1 errors
__SECTION__
compile_Err:

__SECTION__
simulation_Out:

__SECTION__
simulation_Err:
Traceback (most recent call last):
  File "C:\Users\lucco\Desktop\tests\pyLang\compiler\vm_simulator.py", line 17, in <module>
    with open(args.input,'rb') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'tests/tabs_test.hex'